features/step_definitions/installer.rb

```diff
@@ -0,0 +1,24 @@
+## installer related steps such as upgrade
+#
+# NOTE: it is expected that user set the environment variable UPGRADE_TARGET_VERSION prior to running this step
```

Comment:
I don't see this variable used in the step, only in scenarios. This makes sense to me. But this note here would be confusing. Better remove that note and add list of step table parameters that need to be added instead.